crates/uv/tests/help.rs

```diff
@@ -225,9 +230,11 @@ fn help_subcommand() {
     success: true
     exit_code: 0
     ----- stdout -----
+
+    ----- stderr -----
     Manage Python installations
 
-    Usage: uv python [OPTIONS] <COMMAND>
+    Usage: python <COMMAND>
```

Comment:
This should keep the uv preview

Reply:
Will investigate. I don't know how Clap renders these things in a parent-aware style.

Reply:
Fixed.